libs/template/presets/storefront/app/stable/experience/experience-data.ts

```diff
@@ -0,0 +1,39 @@
+import { AppFeature } from '@spryker-oryx/core';
```

Comment:
As before, I'd try to avoid this much of code duplication. 
If latest has no difference with current/stable, I'd just reuse stable. 

In case of static data, we may think about additional providers for latest, that could even just use static data merging strategies (on top of stable data).